README.md

```diff
@@ -58,16 +76,15 @@ See [here](https://docs.ansible.com/ansible/devel/dev_guide/developing_collectio
 
 ### Communication
 
-We have a dedicated Working Group for Ansible development.
+We announce important development changes and releases through the [Ansible Bullhorn newsletter](http://eepurl.com/gZmiEP). If you are a collection developer, be sure you are subscribed.
+
+Join us in the ``#ansible-community``and other [IRC channels](https://docs.ansible.com/ansible/devel/community/communication.html#irc-channels).
```

Comment:
Hmm, shouldn't we also mention #ansible for support? Otherwise we might get more visitors asking for support that need to be redirected.

Reply:
Sounds sensible, added, thanks